samples/subsys/usb/webusb/README.rst

```diff
@@ -53,35 +52,37 @@ web page.
 
 Follow below steps to run the demo on host:
 
-1. Run the latest dev-channel release of Google Chrome on host.
-   https://www.google.com/chrome/browser/desktop/index.html?extra=devchannel
+#. Run the latest Google Chrome on host.
```

Comment:
Is there a minimum version number needed?  If so, you should mention that.

Reply:
I do not know from which version Chrome supports this API. At the time of writing Chrome downloaded from official site works.